FlowCal/excel_ui.py

```diff
@@ -133,7 +141,8 @@ def read_table(filename, sheetname, index_col=None):
 
     """
     # Catch sheetname as list or None
-    if sheetname is None or hasattr(sheetname, '__iter__'):
+    if sheetname is None or \
+        (hasattr(sheetname, '__iter__') and not isinstance(sheetname, str)):
```

Comment:
I read more into it, and `six` seems to be a better way to solve this. Alternatively, I could just copy the relevant portion of code:

```python
PY3 = sys.version_info[0] == 3

if PY3:
    string_types = str,
else:
    string_types = basestring
```

I like the second better, since I wouldn't have to worry about a new dependency. Although `six` seems to be included by default in Anaconda, and in a python3.6 virtual machine set up with `pip`.